mkdocs.yml

```diff
@@ -433,6 +433,7 @@ nav:
               - 'M.2 Wi-fi': unified-test-documentation/dasharo-stability/C02-m2-wi-fi.md
               - 'NVMe detection': unified-test-documentation/dasharo-stability/C03-nvme-detection.md
               - 'NET interface check after coldboot/warmboot/reboot/suspend': unified-test-documentation/dasharo-stability/01-net-controller-after-coldboot-warmboot-reboot-suspend.md
+              - 'Battery thresholds': unified-test-documentation/dasharo-stabilityC04-battery-thresholds.md
```

Comment:
```suggestion
              - 'Battery thresholds': unified-test-documentation/dasharo-stability/C04-battery-thresholds.md
```

Reply:
should fix the `Check orphan pages` CI check.

Reply:
fixed in https://github.com/Dasharo/docs/pull/623/commits/607ae275ca6b7c5fcb0edec9df6bf05fb8f3b6db